x-pack/plugins/security_solution/common/detection_engine/prebuilt_rules/api/get_prebuilt_rules_status/response_schema.ts

```diff
@@ -6,12 +6,8 @@
  */
 
 export interface GetPrebuiltRulesStatusResponseBody {
-  status_code: number;
-  message: string;
-  attributes: {
-    /** Aggregated info about all prebuilt rules */
-    stats: PrebuiltRulesStatusStats;
-  };
+  /** Aggregated info about all prebuilt rules */
+  stats: PrebuiltRulesStatusStats;
```

Comment:
Is the endpoint open to returning other kind of information other than `stats` in the future; do we have any future use cases for that?

Just wandering why we would hold all data within the single `stats` property instead of making the stats themselves first-class properties.

Reply:
Currently, I do not anticipate extending the endpoint to return information other than `stats`. However, it's always good practice to keep future use cases in mind and design with flexibility for potential changes. If a new use case does arise in the future, it would be pretty straightforward to adapt this endpoint.